bindings/python/src/expose-solvers-base.cpp

```diff
@@ -11,17 +11,6 @@ using VParams = ValueFunctionTpl<Scalar>;
 } // namespace python
 } // namespace aligator
 
-namespace eigenpy {
-namespace internal {
-
-template <>
-struct has_operator_equal<::aligator::python::QParams> : boost::false_type {};
-template <>
-struct has_operator_equal<::aligator::python::VParams> : boost::false_type {};
-
-} // namespace internal
-} // namespace eigenpy
-
```

Comment:
Instead of removing this code, you can use the #if EIGENPY_VERSION_AT_MOST(3.2.0) to be compatible with oldest EigenPy version.

Reply:
Ah clever, I'll do that

Reply:
Done